rplugin/python3/LanguageClient/LanguageClient.py

```diff
@@ -190,19 +190,46 @@ def start(self) -> None:
 
         self.defineSigns()
 
-        try:
-            self.nvim.call('cm#register_source', dict(
-                name='LanguageClient_%s' % languageId,
-                priority=9,
-                scopes=[languageId],
-                abbreviation='',
-                cm_refresh='LanguageClient_completionManager_refresh'))
-        except Exception as ex:
-            logger.warn("register completion manager source failed.")
-
         logger.info('End LanguageClientStart')
 
-        self.initialize([])
+        def cb(result: Dict):
```

Comment:
This is intentional, 
```
Note: place the callback here to capture the languageId, which is
the `filetype`. Otherwise it will fail if the user change the
buffer before this callback is called.
```

If I place it in other functions, then I have to get the languageId again, which may not be the same languageId if the user switch buffer befure that.